.github/workflows/build.yml

```diff
@@ -322,26 +322,26 @@ jobs:
       uses: actions/upload-artifact@v4
       if: startsWith(matrix.os, 'macos') && startsWith(matrix.runtime, 'osx-x64')
       with:
-        name: freetube-${{ steps.versionNumber.outputs.result }}-mac-x64.zip
+        name: freetube-${{ steps.versionNumber.outputs.result }}-mac-x64-portable.zip
```

Comment:
Not objecting this but I found that projects rarely got portable suffix to zips & dmg for mac
But it's still fine to be extra clear

Reply:
I can revert it if you want. I only added this because some of our portable builds did have it and others not

Reply:
Ya I think it's better to revert.
There is no portable concept for macOS apps (mostly). The data is stored in user directory anyway.
I would even argue that the windows one isn't portable as well. Since the data isn't stored inside the same folder as the app folder (and no way to change that). It's just another way of installation but the data isn't "portable".

Reply:
> I would even argue that the windows one isn't portable as well. Since the data isn't stored inside the same folder as the app folder (and no way to change that). It's just another way of installation but the data isn't "portable".

A conversation i had before opening this PR

![image](https://github.com/user-attachments/assets/565ba80a-89b3-44b9-bfe5-919e759f7f56)